recipes/squeakr/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "squeakr" %}
+{% set version = "0.5" %}
+{% set md5 = "311dd917960e91e4fb7fee34970cd0b5" %}
+
+package:
+  name : {{ name|lower }}
+  version : {{ version }}
+
+source:
+  url : https://github.com/splatlab/squeakr/archive/v{{ version }}.tar.gz
+  fn : {{ name|lower }}_{{ version }}.tar.gz
+  md5 : {{ md5 }}
+
+build:
+  number : 1
+
+requirements:
+  build:
+    - gcc  # [not osx]
+    - llvm  # [osx]
+    - bzip2
+    - zlib
+    - boost 
+    - openssl
+  run:
+    - bzip2
```

Comment:
Any cou probably want to pin zlib, boost and bzip2 as well.

Reply:
What's the easiest way to do that --- is there a "current version" variable I can use, or should I hard code the versions?

Reply:
```
- zlib {{CONDA_ZLIB}}*
```
https://github.com/bioconda/bioconda-recipes/blob/b1a58891842d9f7f7945b20f5d4701a435879a51/recipes/pybedtools/0.6.9/meta.yaml

and boost:
`- boost {{CONDA_BOOST}}*`

Reply:
I will include a variable for bzip2 as well.